build.sbt

```diff
@@ -56,6 +56,7 @@ lazy val docs =
     .settings(
       scalaVersion := "2.13.6",
       mdocOut := (ThisBuild / baseDirectory).value
+      mdocAutoDependency := false
```

Comment:
It is missing a comma in the previous line

Reply:
Stupid mistake, thx